.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -19,6 +19,33 @@ jobs:
       - uses: actions/checkout@v4
       - uses: ./.github/actions/pnpm
 
+  fetch-baseline-screenshots:
```

Comment:
Moved this up near to `install-dependencies` because they're the first two jobs to run.